modules/i3s/src/helpers/i3s-nodepages-tiles.js

```diff
@@ -0,0 +1,97 @@
+import {load} from '@loaders.gl/core';
+import {normalizeTileNonUrlData} from '../lib/parsers/parse-i3s';
+
+export default class I3SNodePagesTiles {
```

Comment:
This name is not very clear to me. Would be good to have some JSDoc for the class and methods. Maybe here or in separate typescript files.

Reply:
Done